quadratic_equation.py

```diff
@@ -0,0 +1,54 @@
+import math
+
+def get_numbers_input() -> tuple[str]:
+    """
+    getting from terminal data and returning it
+    :return: tuple of three strings separated by spaces
+    """
+    return input("Enter tuple of three digits by space: ").split()

```

Comment:
but you returning list, not tuple. pls convert this to tuple

Reply:
fixed